README.md

```diff
@@ -4,10 +4,30 @@ Welcome to the Chengdu chapter of [ProtoSchool](https://proto.school)!
 
 We are a group of people passionate about the decentralized web and excited to learn, teach, and share ideas. Everyone is welcome!
 
+We focus on building IPFS ecosystem (Decentralized Storage Network) cooperate with [Protocol Labs](https://protocol.ai/).  
+
+Everyone is welcome to join us to learn, to preach, to research, to develop, to share ideas of the amazing tech together!   
```

Comment:
```suggestion
Everyone is welcome to join us to learn, to promote, to research, to develop, and to share ideas of the amazing tech together!   
```

Reply:
I'm suggesting this change to remove the religious connotation and make more people feel welcome. :)